src/monodroid/jni/dylib-mono.cc

```diff
@@ -660,40 +664,32 @@ DylibMono::object_unbox (MonoObject *obj)
 	return mono_object_unbox (obj);
 }
 
-void
-DylibMono::profiler_install (void *profiler, void *callback)
-{
-	if (mono_profiler_install == nullptr)
-		return;
-
-	mono_profiler_install (profiler, callback);
-}
-
-void
-DylibMono::profiler_install_jit_end (MonoProfileJitResult end)
+MonoProfilerHandle
+DylibMono::profiler_create ()
 {
-	if (mono_profiler_install_jit_end == nullptr)
-		return;
+	if (mono_profiler_create == nullptr)
+		return NULL;
```

Comment:
please use `nullptr` here and elsewhere - this is a strongly typed version of `null` and should always be used instead of the old macro.

Reply:
Updated.